src/util/ses.ts

```diff
@@ -378,32 +386,28 @@ export function sendNFTBookPendingClaimEmail({
           Charset: 'UTF-8',
           Data: getNFTTwoContentWithMessageAndButtonTemplate({
             title1: titleZh,
-            content1: `<p>親愛的讀者：</p>
-            <p>感謝支持並購買<a href="${nftPageURLZh}">《${bookName}》</a>。</p>
-            <p>請根據網頁指示領取你的電子書，亦可按以下連結前往該頁面。完成步驟後，即可領取電子書。</p>
-            <p>若你購買的電子書需要作者親自簽發，請在完成步驟後，耐心等待作者發貨。
+            content1: `<p>親愛的${receiverDisplayName || '讀者'}：</p>
+            <p>感謝購買<a href="${nftPageURLZh}">《${bookName}》</a>。</p>
+            <p>請<a href="${claimPageURLZh}">登入 Liker Land 書架</a>閱讀你的電子書。
+            若你未註冊 Liker Land 帳號，請點擊<a href="${claimPageURLZh}">這裡</a>註冊。</p>
+            <p>若你購買的電子書需要作者親自簽發，請先點擊<a href="${claimPageURLZh}">這裡</a>登入 Liker Land，然後耐心等待作者的發貨通知。
             作者會在 1-3 個工作天內親手簽發你的電子書。
-            屆時請往你的 <a href="${portfolioURLZh}">Liker Land 個人主頁</a>查閱。</p>`,
-            buttonText1: '領取我的電子書',
-            buttonHref1: claimPageURLZh,
+            屆時請往你的 <a href="${portfolioURLZh}">Liker Land 書架</a>查閱。</p>`,
             append1: `<p>如有任何疑問，歡迎<a href="${CUSTOMER_SERVICE_URL}">聯絡客服</a>查詢。
             <br>感謝珍藏此書，願你享受閱讀的樂趣。</p>
             <p>Liker Land</p>
             <p>[${from}]</p>`,
 
             // English version
             title2: titleEn,
-            content2: `<p>Dear reader,</p>
+            content2: `<p>Dear ${receiverDisplayName || 'reader'},</p>
             <p>Thanks for purchasing "<a href="${nftPageURLEn}">${bookName}</a>".</p>
-            <p>Please follow the steps on the web to claim your ebook.
-            You can also click the button below to get a direct link to that page.
-            Once the steps are completed, you can receive your ebook.</p>
-            <p>If the ebook you purchased requires a personal signature from the author,
-            please wait patiently for the author to dispatch it after completing the steps,
-            typically within 3 days.
-            You can check the status of the ebook on your <a href="${portfolioURLEn}">Liker Land Dashboard</a>.</p>`,
-            buttonText2: 'Claim your ebook',
-            buttonHref2: claimPageURLEn,
+            <p>Please <a href="${claimPageURLEn}">login to Liker Land Bookshelf</a> to read your e-book.
+            If you have not registered a Liker Land account, please <a href="${claimPageURLEn}">click here</a> to register.</p>
+            <p>If the e-book you purchased requires the author's personal signature,
+            please first log in to Liker Land by clicking here and then patiently wait for the author's dispatch notification.
```

Comment:
```suggestion
            please first log in to Liker Land and register by clicking here, and then patiently wait for the author's dispatch notification.
```

Since reader still need to claim the book for author to send, I suggest some kind of action should be hinted, otherwise registered user will misunderstand no action has to be taken